README.md

```diff
@@ -1,5 +1,12 @@
 # Tekton Hub 
 
+> [!IMPORTANT]
+> **The Tekon Hub project is deprecated.**
+>
+> - On September 1st, 2025, the public instance (https://hub.tekton.dev) will be shut-down.
+> - On November 1st, 2025, this repository will be archived (and thus frozen).
+
```

Comment:
Should we add the Artifact Hub references here as an alternative and might be useful for users?